server/seeder/index.js

```diff
@@ -0,0 +1,51 @@
+import express from 'express';
+import redis from 'redis';
+import client from '../helpers/redis';
+import * as helpers from './helper';
+import data from './placements';
+
+const router = express.Router();
+
+router.get('/place', (req, res) => {
+  client.set('allocations', JSON.stringify(data), redis.print);
+  res.send(data);
+});
+
+const possiblePlacementStatus = [
+  'Placed - Awaiting Onboarding',
+  'External Engagements - Rolling Off',
+];
+
+const placements = () => new Promise((resolve, reject) => {
+  client.get('allocations', (error, result) => {
+    if (error) {
+      console.log(error);
+      reject(error);
+    }
+    return resolve(result);
+  });
+});
+
+const pipeline = async (req, res) => {
+  let values = await placements();
+  if (!values) {
+    values = await placements();
```

Comment:
Why the repeated call to placements()?

Reply:
I was trying out a concept and I forgot to get rid of it.